src/foam/u2/view/UnstyledTableView.js

```diff
@@ -251,6 +252,15 @@ foam.CLASS({
     {
       name: 'dataChangedFlag',
       class: 'Boolean'
+    },
+    {
+      name: 'subStack',
+      transient: true,
+      flags: ['js'],
```

Comment:
Why is this needed?

Reply:
I mean flags:

Reply:
fixed, all of those changes were needed for ApprovalRequest java class